### DISCUSSION TITLE
Workspace UX enhancement

### DISCUSSION THREAD

**Discussion by LucasM314** [zen-browser/desktop#3204] [Ideas]
Hey!

First, thanks for the hard work, the browser offers so many useful features already, I love it!
Overall, I'd say the UI is clear, with one exception: **workspaces**. It took me an _hour_ to figure out how to create a new workspace, since right-clicking on a text label is definitely not the most intuitive UX!

I think there should be a "+" button on the bottom right of the sidebar allowing to create a new workspace.
Since both bottom corners are already used, I see two possible solutions:
 - The profile icon could be moved to the top left (or right)
 - The download icon could be moved to the top right

In my opinion, the second one makes more sense, since it allows the bottom of the sidebar to feature the two concepts used to organise the browser experience: profiles and containers.

That being said, awesome work again!

**Comment by 12th-devs** [comment]
I just figured I would note, they mean the default fonticon locations, as they are customizable

**Reply by LucasM314** [reply]
Indeed, the profile and download fonticons positions are customisable through the Customize Toolbar option in the context menu. However, I don't know how to add a "+" button whose purpose would be to open the workspace creation wizard.

**Reply by 12th-devs** [reply]
Yes, this would be a good idea. Or something in the original onboarding along with the container setup
